components/textfield/metadata/mods.md

```diff
@@ -1,5 +1,6 @@
 | Modifiable Custom Properties                            |
 | ------------------------------------------------------- |
+| `--mod--textfield-icon-spacing-inline-start-invalid`    |
```

Comment:
It looks like this mod name in the CSS just needs a dash removed (and the mods.md regenerated).

Reply:
good catch! Fixed!